pyproject.toml

```diff
@@ -34,22 +34,22 @@ Documentation = "https://ccao-data.github.io/assesspy/"
 
 [project.optional-dependencies]
 dev = [
-  "setuptools >= 61.0",
-  "pytest >= 7.3.0",
-  "pytest-cov >= 4.1.0",
+  "setuptools>=61.0",
+  "pytest>=7.0.0",
```

Comment:
**[Question, non-blocking]** Oops, just noticed this -- it seems like the `minversion` is set to 6.0 in the pytest config in `pyproject.toml`, but 7.0 here. Not a big discrepancy, and I doubt it matters much, but I'm curious which one of these is the actual minimum version?

Reply:
I think we use some functionality from 7.0.0, but I can't recall what. I'll just set it to 7.0.0 for the sake of safety.